src/ServiceControl/Operations/ErrorQueueImport.cs

```diff
@@ -137,14 +137,16 @@ private void InnerHandle(TransportMessage message)
 
             var failureDetails = failedMessageFactory.ParseFailureDetails(message.Headers);
 
+            var replyToAddress = message.ReplyToAddress == null ? null : message.ReplyToAddress.ToString();
```

Comment:
You can safely ignore this value. It used to be used to preserve the value when the underlying transport core did not do so. It should eventually be removed.

The main issue is [here](https://github.com/Particular/ServiceControl/blob/d78e169da41f38e54645b254465a40dcd0ad6d09/src/ServiceControl/Recoverability/Retries/RetryProcessor.cs#L211). This transport message contains the unadulterated ReplyToAddress header but the transport will run it through `Address.Parse().ToString()` before writing it to the table as outlined in https://github.com/Particular/NServiceBus.SqlServer/issues/345